package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "google-maps",
   "widgetName": "GoogleMaps",
-  "version": "1.4.3",
+  "version": "1.5.0",
```

Comment:
please make sure that the `package.xml` is also updated

Reply:
Done